wallet/src/util/keyManagement.ts

```diff
@@ -403,6 +404,7 @@ export const makeInteractiveSigner = async (
   const signingClient = await connectWithSigner(chainInfo.rpc, offlineSigner, {
     aminoTypes: new AminoTypes(converters),
     registry: SwingsetRegistry,
+    accountParser: accountFromAny,
```

Comment:
I don't anticipate this is strictly necessary because this client is used for signing agoric transactions, and only the one in `ibcTransfer.ts` signs stride messages, but might as well be safe.